buildSrc/src/main/groovy/bosk.maven-publish.gradle

```diff
@@ -73,8 +73,8 @@ publishing {
 			url version.endsWith('SNAPSHOT') ? snapshotsRepoUrl : releasesRepoUrl
 
 			credentials {
-				username ossrhUsername
-				password ossrhPassword
+				ext.username = project.properties['ossrhUsername'] ?: 'bogusUser'
+				ext.password = project.properties['ossrhPassword'] ?: 'bogusPassword'
```

Comment:
What does the `ext.` prefix do?

Reply:
It's the gradle `[ExtraPropertiesExtension](https://docs.gradle.org/current/dsl/org.gradle.api.plugins.ExtraPropertiesExtension.html)`

Reply:
```
Extra properties extensions allow new properties to be added to existing domain objects. They act like maps, allowing the storage of arbitrary key/value pairs. All [ExtensionAware](https://docs.gradle.org/current/dsl/org.gradle.api.plugins.ExtensionAware.html) Gradle domain objects intrinsically have an extension named “ext” of this type.
```

Reply:
Ok... so the `credentials` object is "extension aware" and so you can put the credentials on `ext` instead of putting them directly on the `credentials` object?

Reply:
Actually it looks like we might not need to use the extension at all:
https://github.com/venasolutions/bosk/compare/3da65c2c8578e797be6345601e28933c77402fe8..d655002dee6443191551ff5fb7b74408df33c7a1